Core/PaymentMethodHelper.php

```diff
@@ -70,17 +72,152 @@ public static function getModulePayments()
     }
 
     /**
-     * Generates a Mandate for SEPA transactions
+    * Generates a Mandate for SEPA transactions
+    *
+    * @param int $iOrderNumber
+    *
+    * @return Mandate
+    *
+    * @since 1.0.1
+    */
+    public static function getMandate($iOrderNumber)
+    {
+        $iLength = self::MAX_MANDATE_ID_LENGTH - 1 - strlen((string) time());
+        return new Mandate(substr($iOrderNumber, 0, $iLength) . '-' . time());
+    }
+
+    /**
+     * Returns account holder for Sepa Direct Debit
```

Comment:
Let's please use the term "SEPA" consistently. Currently, it is called "sepa"/"Sepa"/"SEPA".